python/ray/data/_internal/execution/operators/actor_pool_map_operator.py

```diff
@@ -430,10 +433,27 @@ class _ActorPool:
     This class is in charge of tracking the number of in-flight tasks per actor,
     providing the least heavily loaded actor to the operator, and killing idle
     actors when the operator is done submitting work to the pool.
+
+    Args:
+        max_tasks_in_flight: The maximum number of tasks to queue on a single actor at
+            a time. The actual max_tasks_in_flight starts at 1, and then ramps up
+            to this value after `ramp_up_threshold` actors have been created, to allow
+            for even scheduling for datasets with a small number of partitions.
+        max_tasks_rampup_threshold: The number of actors to wait for before ramping up
+            `max_tasks_in_flight`.
     """
 
-    def __init__(self, max_tasks_in_flight: int = float("inf")):
-        self._max_tasks_in_flight = max_tasks_in_flight
+    def __init__(
+        self,
+        max_tasks_in_flight: int = float("inf"),
+        max_tasks_rampup_threshold: int = 1,
+    ):
+        # We start with 1 task in flight, then ramp up to `max_tasks_in_flight`
+        # after the `max_tasks_rampup_threshold` actors have been created.
```

Comment:
Could you leave a comment of the motivation? So people in the future read it would understand why we need to do it. Just copy some of explanation from PR description is good enough. Thanks.

Reply:
added